docs/docs/spark-procedures.md

```diff
@@ -976,18 +976,18 @@ CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id =>
 
 ## Table Replication
 
-The `rewrite-table-path` assists in moving or copying an Iceberg table from one location to another.
+The `rewrite-table-path` procedure prepares an Iceberg table for moving or copying to another.
```

Comment:
Nit: add location at the end of the sentence

Reply:
ops, copy paste mistake